docs/Q-and-A.md

```diff
@@ -0,0 +1,746 @@
+{TBW}
+## Does a Sidetree wallet store virtual credentials connect to my identifiers?
+{TBW}
+
+# Q&A section Signatures
+
+## Who can sign off my proofs and identifiers?
+{TBW}
+## What is the practical use of signatures?
+{TBW}
+
+# Q&A section Proofs
+
+## What does Sidetree proof?
+{TBW}
+
+## Do I need to show the full IPFS Sidetree database to anybody I transact with, even though I'd only like to show a part of it, for example a virtual credential?
+{TBW}
+
+# Q&A section Private Key Management
+
+## How secure is the Sidetree infrastructure?
+{TBW}
+
+## How multi-tasking is Sidetree?
+{TBW}
+## Does your public-private-key format matter in Sidetree?
+{TBW}
+
+## Not your keys, not your identity?
+{TBW}
+
+## The wallet is there to store my Sidetree private keys safely, no?
+{TBW}
+## Are compound private keys (Shamir Secret Sharing) and multisignature schemes possible to create identifiers?
+{TBW}
+## How to delegate control over my private keys that control my identifiers?
+{TBW}
+# Q&A section Blockchain
+
+## what does an ID system gain from a blockchain?
+I would actually harken back to what Satoshi first called Bitcoin before it was blockchain, which was timechain. It turns out that `PKI`, which is public key infrastructure and that's like the thing that backs identifiers. I'll give you an analogy of what one is today. An existing example is the DNS system.
+
+The identifier is a domain name and then the backing PKI data is your zone file in DNS that lists your named servers, routing information, then there's the certificate of authorities that have the sort of the backing cryptography. So that's an example of a PKI system. Every PKI system, if it isn't centralized, like a system where they just kind of decree that this users is usurped, if you want to decentralize that, the most important piece is having a global immutable append-only log, because everyone needs to see the state of some identifier. Like if Alice creates an ID and she initially associates, say her phone that she has at the time with her ID and a public key on it, when she goes to get a new phone, she's going to switch that key up.
+
+So everyone needs to be able to see that event, globally and deterministically, or else someone could masquerade as Alice. They could say, "Well, I have her old key so I'm her." You have to have this sort of linear chronology of what happened when, it's a state machine. So what a blockchain does, really elegantly that we never had before, is it's an oracle for entering events into a chronology. That's exactly what we use it for. In its most primitive sense, we use Bitcoin to enter in events in a chronology that nodes sort of watch for and compute so they all can understand the state of every ID that's anchored in Bitcoin. So it's very important.\
+[Source: What bitocin Did 231](https://www.whatbitcoindid.com/wbd231-daniel-buchner)
+
+## So why would we pick Bitcoin versus some other blockchain, like Ethereum or Hyperledger?
```

Comment:
Presumably this one is an ion-specific Q&A. There are also Sidetree implementations on Ethereum and Hyperledger Fabric.

Reply:
Then I need to adjust the tone of voice, because it's meant as a Sidetree Q-and-A. I have added your input to the Q-and-A.